testprojects/3rdparty/com/google/guava/BUILD

```diff
@@ -0,0 +1,7 @@
+# NB: This is an intentionally ancient guava to attempt to flush out classpath collisions
+# with the compiler.
+jar_library(name='ancient-guava',
```

Comment:
If the guava zinc uses is already shaded, why do we still need to worry about classpath collision for the testproject?

Reply:
We're attempting to _cause_ an incompatible collision here, which might catch bugs (like the lack of shading) in the future.

Reply:
Ah I see that's what you mean by
> flush out classpath collisions

Reply:
It's unclear to me why an ancient guava would flush out classpath problems. Is Zinc using an ancient version?